### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by sivayogasubramanian**
Closes https://github.com/AY2122S1-CS2103T-W16-2/tp/issues/218
Closes https://github.com/AY2122S1-CS2103T-W16-2/tp/issues/169

Can view it [here](https://sivayogasubramanian.github.io/tp/UserGuide.html).

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ features are available in BTBB. For more in depth information on the available f
 If you have any questions regarding BTBB, check out [FAQ](#5-faq) for answers to frequently asked questions.
 
 [Disclaimers](#6-disclaimers) contains general disclaimers for our app such as restrictions on inputs and
-additional details regarding the command format. **It is important to read this section before reading the rest of the
+additional details regarding the command format. **It is important for you to read this section before reading the rest of the
```
Comment (Imranr2) [inline docs/UserGuide.md:35]: I don't think the user should read all of the disclaimers before reading the rest of the UG.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -437,25 +438,25 @@ Format: `edit-c INDEX [cn/NAME] [cp/PHONE] [ce/EMAIL] [ca/ADDRESS]`
...
 
 <div markdown="block" class="alert alert-primary">
 
 **:bookmark: Note:**<br>
 
-* The search is case-insensitive.
+* You should note that the search is case-insensitive.
 
-* There must be 1 or more search arguments.
+* You should supply 1 or more search arguments.
```
Comment (Imranr2) [inline docs/UserGuide.md:451]:
```suggestion
* You must supply 1 or more search arguments.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -796,25 +819,25 @@ Format: `edit-o INDEX [c/INDEX] [cn/CLIENT_NAME] [cp/CLIENT_PHONE] [ca/CLIENT_AD
...
```
Comment (Imranr2) [inline docs/UserGuide.md:832]:
```suggestion
* You must supply 1 or more search arguments.
```

**Review by Imranr2 — CHANGES_REQUESTED** [review]
Minor nits

File: docs/UserGuide.md (hunk 1 above)
Comment (sivayogasubramanian) [reply docs/UserGuide.md:35]: This is intended

File: docs/UserGuide.md [hunk 4]
```diff
@@ -796,25 +819,25 @@ Format: `edit-o INDEX [c/INDEX] [cn/CLIENT_NAME] [cp/CLIENT_PHONE] [ca/CLIENT_AD
...
-* Multiple search keywords can be specified for each field. <br>
+* You can specify multiple search keywords for each field. <br>
   e.g. <code>find-o cn/Alex Brian</code>
 
-* Partial search will be allowed. <br>
+* You should note that partial search is allowed. <br>
   e.g. <code>find-o cn/Al</code> can show orders for clients with names like Alice and Alex.
 
-* It will find orders that match at least one keyword, for each prefix.
+* You should note that this command will find clients that match at least one keyword, for each prefix.
```
Comment (EuniceSim142) [inline docs/UserGuide.md:840]:
```suggestion
* You should note that this command will find orders that match at least one keyword, for each prefix.
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -858,7 +881,7 @@ Format: `done-o INDEX`
 
 #### 4.5.9 Marking an order as undone: `undone-o`
 
-Marks order as undone.
+You can mark order as undone using this command.
```
Comment (EuniceSim142) [inline docs/UserGuide.md:884]:
```suggestion
You can mark an order as undone using this command.
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -835,13 +858,13 @@ Format: `find-o [cn/CLIENT_NAME] [cp/CLIENT_PHONE] [ca/CLIENT_ADDRESS] [rn/RECIP
...
 
-Lists all orders in the application.
+You can list all orders in the application using this command.
 
 Format: `list-o`
 
 #### 4.5.8 Marking an order as done: `done-o`
 
-Marks order as done once it has been delivered to the client.
+You can mark order as done once it has been delivered to the client using this command.
```
Comment (EuniceSim142) [inline docs/UserGuide.md:867]:
```suggestion
You can mark an order as done once it has been delivered to the client using this command.
```

**Review by EuniceSim142 — COMMENTED** [review]
LGTM, just some minor nits.

**Review by EuniceSim142 — APPROVED** [review]
LGTM, thanks!

**Review by Imranr2 — APPROVED** [review]
LGTM!

**Pull request merged**